pkg/apis/stork/v1alpha1/snapshotrestore.go

```diff
@@ -29,18 +27,34 @@ type VolumeSnapshotRestoreSpec struct {
 type SnapshotRestoreStatusType string
 
 const (
-	// SnapshotRestoreStatusInitial is the initial state when snapshot restore is initiated
-	SnapshotRestoreStatusInitial SnapshotRestoreStatusType = ""
-	// SnapshotRestoreStatusPending for when restore is in pending state
-	SnapshotRestoreStatusPending SnapshotRestoreStatusType = "Pending"
-	// SnapshotRestoreStatusReady for when restore is done
-	SnapshotRestoreStatusReady SnapshotRestoreStatusType = "Ready"
-	// SnapshotRestoreStatusError for when restore failed
-	SnapshotRestoreStatusError SnapshotRestoreStatusType = "Error"
+	// VolumeSnapshotRestoreStatusInitial is the initial state when snapshot restore is initiated
+	VolumeSnapshotRestoreStatusInitial SnapshotRestoreStatusType = ""
+	// VolumeSnapshotRestoreStatusPending for when restore is in pending state
+	VolumeSnapshotRestoreStatusPending SnapshotRestoreStatusType = "Pending"
+	// VolumeSnapshotRestoreStatusReady for when restore is done
+	VolumeSnapshotRestoreStatusReady SnapshotRestoreStatusType = "Ready"
+	// VolumeSnapshotRestoreStatusError for when restore failed
+	VolumeSnapshotRestoreStatusError SnapshotRestoreStatusType = "Error"
+)
+
+// VolumeSnapshotRestoreStageType is stage of volume in-place restore
+type VolumeSnapshotRestoreStageType string
+
+const (
+	// VolumeSnapshotRestoreStageInitial is the initial stage when snapshot restore is initiated
+	VolumeSnapshotRestoreStageInitial VolumeSnapshotRestoreStageType = ""
```

Comment:
The stage and status have same values. Maybe we don't need stage for this. Just the following status: Initial, Pending, InProgress, Failed, Successful